packages/clay-date-picker/src/__tests__/IncrementalInteractions.tsx

```diff
@@ -11,6 +11,7 @@ import React from 'react';
 const spritemap = 'icons.svg';
```

Comment:
It worries me a bit that we had to update these tests. Was this done for locale or is there a particular reason we had to do so?

Reply:
As the spritemap will have no effect in tests and the components need this information, so we pass on any value.

Reply:
@matuzalemsteles sorry, I meant for this whole file. I just left the comment at the first line I saw in the file

Reply:
Oh okay 😅. Are you trying to tell if these changes in testing make sense? I think the last time Diego talked about it was because he made the changes to the `aria-label` of `<DayNumber />` to be more accessible.

Reply:
ah I see. Yeah I was just making sure that we aren't introducing any breaking changes for people. Typically I check out tests because if we require any changes to how we test, it sometimes indicates that users will also have to make a change

Reply:
Yeah I totally agree